persist/pom.xml

```diff
@@ -12,7 +12,7 @@
   <properties>
     <flyway.version>6.0.3</flyway.version>
     <database.driver>org.postgresql.Driver</database.driver>
-    <database.url>jdbc:postgresql://localhost:5432/c4cneu-db?autoreconnect=true</database.url>
+    <database.url>jdbc:postgresql://localhost:5434/c4cneu-db?autoreconnect=true</database.url>
```

Comment:
Sorry you had to change all of these occurances! We should have an environment variable file for this to make development easier.